client/app/widgets/Comments/__tests__/Comments.spec.jsx

```diff
@@ -83,41 +80,33 @@ const component = <Comments formProps={formProps} />;
 
 describe('Comments', () => {
   beforeEach(() => {
-    axiosPostSpy = jest.spyOn(axios, 'post').mockImplementation(() => Promise.resolve({
+    jest.spyOn(axios, 'post').mockResolvedValue({
       data: { comment },
-    }));
-    axiosDeleteSpy = jest.spyOn(axios, 'delete').mockImplementation(() => Promise.resolve({
+    });
+    jest.spyOn(axios, 'delete').mockResolvedValue({
       data: { id },
-    }));
+    });
   });
 
   it('renders correctly', () => {
-    let wrapper = null;
-    expect(() => {
-      wrapper = mount(component);
-    }).not.toThrow();
-    expect(wrapper).not.toBeNull();
+    render(component);
+    expect(screen.queryByRole('article')).not.toBeInTheDocument();
```

Comment:
My mistake. I didn't see earlier that this [component also renders](https://github.com/ifmeorg/ifme/blob/main/client/app/widgets/Comments/index.jsx#L135) a `section` around all the comments, which would make sense to check too. But I think checking for an `article` also works 👍

Reply:
I'll keep it as is.